myhdl/_ShadowSignal.py

```diff
@@ -244,13 +244,15 @@ def _resolve(self):
     def toVerilog(self):
         lines = []
         for d in self._drivers:
-            lines.append("assign %s = %s;" % (self._name, d._name))
+            if d._driven:
+                lines.append("assign %s = %s;" % (self._name, d._name))
```

Comment:
Not sure I 100% understand these changes.  Do you have an example what this mod is fixing?  Is it covered in the test you created?